base_kanban_stage_state/static/description/icon.svg

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Remove the svg icon, we only need to provide the SVG if we edited or created. It's so that future maintainers can quickly make edits if need be. We have the SVG of this icon in the template module though, so we can save diff space.